contrib/kube-prometheus/hack/grafana-dashboards-configmap-generator/bin/grafana_dashboards_generate.sh

```diff
@@ -0,0 +1,317 @@
+#!/bin/bash
```

Comment:
@gianrubio : Thanks for your comment, I will do the changes next week and update the PR, this week I will be away on vacations.

Also i'm thinking on doing more small changes to have the same philosophy as the rest of your hack/scripts scripts... for example now i'm logging some lines with messages about the processed dashboards and the amount of configmaps generated due to the size limit.

Do you want me to remove the output? (if you have a minute just run `make generate` with my branch to take a look).

The reason of not using `set -e` was also that my script is full of preliminary verifications before the payload.